sources/controllers/Notify.controller.php

```diff
@@ -29,6 +29,7 @@ public function pre_dispatch()
 	{
 		// Our topic functions are here
 		require_once(SUBSDIR . '/Topic.subs.php');
+		require_once(SUBSDIR . '/Boards.subs.php');
```

Comment:
I wonder when we will not have to deal with requires anymore. :laughing: 
/me dreams of autoloading everything. :sleeping_bed:

Reply:
Yeup  ... mysql no longer provides a 5,7 packages for the trusty build.  

I've set things to only use mysq5.6 for the 1.1 branch as I don't want to do the full xenial update that I did for the 2.0 branch.  Also dropped the 5.4 testing since who cares about that.